dashboard/test/ui/runner.rb

```diff
@@ -283,12 +283,19 @@ def output_synopsis(output_text)
     end
   end
 
-  if succeeded
-    HipChat.log "<b>dashboard</b> UI tests passed with <b>#{test_run_string}</b> (#{format_duration(test_duration)})"
+  parsed_output = output_stdout.match(/^(?<scenarios>\d+) scenarios?( \((?<info>.*?)\))?/)
```

Comment:
If this returns nil (because cucumber crashes before it knows anything about scenarios? I don't know cucumber enough to know if this is possible, but might as well be paranoid), we should not try to [] parsed_output below.